actions/workflow.freeipa.user.create.yaml

```diff
@@ -0,0 +1,30 @@
+name: freeipa.user.create
+description: Creates a given list of users
+enabled: true
+entry_point: workflows/freeipa.user.create.yaml
+runner_type: orquesta
+
+parameters:
+  submodule:
+    default: user_delete
+    immutable: true
+    type: string
```

Comment:
This should not be here - its not taken in by the workflow.

Reply:
Should also be `name: workflow.freeipa.user.create` to be consistent with other `workflow.[x]` actions?

Reply:
@ElliottKasoar Yes I was in two minds about it, on the one hand they are all actions that happen to run workflows and freeipa.user.create would probably be the expected name for this but it is inconsistent, so I think your right it should be workflow.* for now at least.